components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -37,26 +54,32 @@ const VulnerabilitiesGrid = ({ onUpdate }) => {
           <div className="govuk-grid-row">
             <div className="govuk-grid-column-one-half">
               <CheckboxList className="vulnerability">
-                {vulnerabilities.map(({ text }, j) => (
-                  <Checkbox
-                    key={`${id}-v-${j}`}
-                    label={text}
-                    name={`${id}-v-${j}`}
-                    onClick={() => updateVulnerabilities(`${id}-v-${j}`, text)}
-                  />
-                ))}
+                {vulnerabilities.map(({ text }, j) => {
+                  return (
+                    <Checkbox
+                      key={`${id}-v-${j}`}
+                      label={text}
+                      name={`${id}-v-${j}`}
+                      onClick={() =>
+                        updateVulnerabilities(`${id}-v-${j}`, text)
+                      }
+                    />
+                  );
+                })}
```

Comment:
Not really a comment, just curious as to why you swapped from `(<... />)` to `{ return (<... />) }`? Was it just a readability thing?

Reply:
Just getting it ready for a change Bogdan is going to make